src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_USERS = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery[] lottery = new Lottery[NUMBER_OF_USERS];
+        for (int i = 0; i < lottery.length; i++) {
+            lottery[i] = new Lottery();
+            System.out.println(lottery[i].getRandomBall());
+        }
```

Comment:
why did you create an array of lotteries? you should create only one lottery instance outside the loop, then instead of array of lotteries you should create an array of balls, inside the loop you should fill your array with random balls and then print all the info about them